src/sentry/static/sentry/app/components/charts/utils.jsx

```diff
@@ -17,15 +23,41 @@ export function truncationFormatter(value, truncate) {
  * Use a shorter interval if the time difference is <= 24 hours.
  */
 export function useShortInterval(datetimeObj) {
-  const {period, start, end} = datetimeObj;
+  const diffInMinutes = getDiffInMinutes(datetimeObj);
 
-  if (typeof period === 'string') {
-    return period.endsWith('h') || period === '1d';
-  }
+  return diffInMinutes <= TWENTY_FOUR_HOURS;
+}
+
+export function getInterval(datetimeObj, highFidelity) {
```

Comment:
isn't `highFidelity=false` implied?

Reply:
It's not required but it can be nice to have it always a boolean and in the method signature so reading the rest of the function won't be required to figure that out